### PR TITLE
Exercise/4.5

### PR DESCRIPTION
**Pull request by DiegoCampos1**

### CODE REVIEW

File: exercises/4_5/index.html [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+      div5.appendChild(img);
+      //8
+      let array = ["um", "dois", "três", "quatro", "cinco", "seis", "sete", "oito", "nove", "dez", ]
+      let lista = document.createElement("ul")
+
+      for (let indice in array){
+      let item = document.createElement("li");
+      item.innerHTML = array[indice];
+      lista.appendChild(item);
+      }
```
Comment (danielPantalena) [inline exercises/4_5/index.html]:
```suggestion
      for (let indice in array) {
        let item = document.createElement("li");
        item.innerHTML = array[indice];
        lista.appendChild(item);
      }
```

File: exercises/4_5/index.html [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+      div6.appendChild(lista);
+      //9 Adicione 3 tags h3, todas sendo filhas do div criado no passo 2.
+      let h3 =[]
+      for (index = 0 ; index < 3 ; index ++){
+      h3[index] = document.createElement("h3");
+      firstDiv.appendChild(h3[index]);
```
Comment (danielPantalena) [inline exercises/4_5/index.html]:
```suggestion
      for (index = 0 ; index < 3 ; index ++){
        h3[index] = document.createElement("h3");
        firstDiv.appendChild(h3[index]);
```

File: exercises/4_5/index.html [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+      }
+      //1 Adicione a classe title na tag h1 criada;
+      title.className = "title 1"
+      //2 Adicione a classe description nas 3 tags h3 criadas;
+     
+      let h=[]
+      for (index = 0 ; index < 3 ; index ++){
+      h = document.querySelectorAll("h3")[index];
+      h.className = "description"
```
Comment (danielPantalena) [inline exercises/4_5/index.html]:
```suggestion
      for (index = 0 ; index < 3 ; index ++){
        h = document.querySelectorAll("h3")[index];
        h.className = "description"
```

File: exercises/4_5/index.html [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+      lista.appendChild(item);
+      }
+      div6.appendChild(lista);
+      //9 Adicione 3 tags h3, todas sendo filhas do div criado no passo 2.
+      let h3 =[]
+      for (index = 0 ; index < 3 ; index ++){
+      h3[index] = document.createElement("h3");
+      firstDiv.appendChild(h3[index]);
+      }
+      //1 Adicione a classe title na tag h1 criada;
```
Comment (danielPantalena) [inline exercises/4_5/index.html]:
```suggestion
      }
      // Segunda Parte
      //1 Adicione a classe title na tag h1 criada;
```

**Review by danielPantalena — APPROVED** [review]
Show!!

**Review by anicioalexandre — COMMENTED** [review]
Boa lógica no ex8! Top!

**Pull request merged**